app/config/index.js

```diff
@@ -1,28 +1,28 @@
-const Joi = require('joi')
-const dbConfig = require('./db')
-const messageQueueConfig = require('./message-queue')
+import joi from 'joi'
+import dbConfig from './db.js'
+import messageQueueConfig from './message-queue.js'
 
-const schema = Joi.object({
-  port: Joi.number(),
-  env: Joi.string().valid('development', 'test', 'production').default('development'),
-  isDev: Joi.boolean().default(false),
-  termsAndConditionsUrl: Joi.string().default('#'),
-  applyServiceUri: Joi.string().default('#'),
-  claimServiceUri: Joi.string().default('#'),
-  endemics: Joi.object({
-    enabled: Joi.boolean().default(false)
+const schema = joi.object({
+  port: joi.number(),
+  env: joi.string().valid('development', 'test', 'production').default('development'),
+  isDev: joi.boolean().default(false),
+  termsAndConditionsUrl: joi.string().default('#'),
+  applyServiceUri: joi.string().default('#'),
+  claimServiceUri: joi.string().default('#'),
+  endemics: joi.object({
+    enabled: joi.boolean().default(false)
   })
 })
 
 const config = {
-  port: process.env.PORT,
+  port: process.env.PORT ? Number(process.env.PORT) : undefined,
   env: process.env.NODE_ENV,
   isDev: process.env.NODE_ENV === 'development',
   termsAndConditionsUrl: process.env.TERMS_AND_CONDITIONS_URL,
   applyServiceUri: process.env.APPLY_SERVICE_URI,
   claimServiceUri: process.env.CLAIM_SERVICE_URI,
   endemics: {
-    enabled: process.env.ENDEMICS_ENABLED
+    enabled: process.env.ENDEMICS_ENABLED && process.env.ENDEMICS_ENABLED === 'true'
```

Comment:
This would convert the value to a boolean `true` if present. But I don't think we actually need this config value for this service do we. Endemics are already enabled so I don't think this app will ever need to concern itself with this flag.

Reply:
Sounds good to me! I'll remove it.